rootfs/rootfs/etc/rc.d/automount

```diff
@@ -34,6 +34,37 @@ if [ ! -n "$BOOT2DOCKER_DATA" ]; then
             mkfs.ext4 -L $LABEL $BOOT2DOCKER_DATA
             swapon "${UNPARTITIONED_HD}2"
         fi
+
+        DISK_VENDOR=$(cat /sys/class/block/$(basename $UNPARTITIONED_HD /dev/)/device/vendor /sys/class/block/$(basename $UNPARTITIONED_HD /dev/)/device/model | tr -d "\n")
+        # Test if disk is "VMware, VMware Virtual S" and empty.
+        if [ "$DISK_VENDOR" = "VMware, VMware Virtual S" ] || [ "$DISK_VENDOR" = "VMware  Virtual disk    " ]; then
```

Comment:
Apparently busybox doesn't support regex in tests, I can't make this to work...

```shell
docker@fusion-test-b2d:~$ cat regextest.sh
#!/bin/sh

DISK_VENDOR="asdasd VMware qweqweqwe"

if [[ "$DISK_VENDOR" == *VMware* ]]; then
        echo "yay"
else
        echo "nay"
fi

docker@fusion-test-b2d:~$ ./regextest.sh
nay
docker@fusion-test-b2d:~$
```

We should be fine with the dual test for now, I guess.